src/types/index.ts

```diff
@@ -68,6 +68,7 @@ export type SemanticDatepickerProps = PickedDayzedProps &
     clearIcon?: SemanticICONS | React.ReactElement;
     filterDate: (date: Date) => boolean;
     format: string;
+    formatOptions: Object | null;
```

Comment:
Please copy the type from [`date-fns`](https://github.com/date-fns/date-fns/blob/4544f0cb1d724bfa7ee846b762bacb8fc303a961/typings.d.ts#L373-L379) and make the prop optional instead of accepting `null`.

Or better, make it an internal type so we can reuse it on the `src/utils.ts` file.

```suggestion
    formatOptions?: {
      locale?: Locale
      weekStartsOn?: 0 | 1 | 2 | 3 | 4 | 5 | 6
      firstWeekContainsDate?: number
      useAdditionalWeekYearTokens?: boolean
      useAdditionalDayOfYearTokens?: boolean
    };
```

Reply:
Done